### ISSUE TITLE
Petraptor Egg recipe can only use small egg

### ISSUE THREAD

**Issue by ubertpendragon** [Regalis11/Barotrauma#10794]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

There are 3 item variants of mudraptor eggs: small, medium and large

It's only possible to craft petraptor egg using the small variant, but you can still pick up medium and large eggs from abandoned outposts.

_This might not be intended as you're not able to pick up petraptor eggs, you can only drop them from the fabricator_

To fix it, you can add the "mudraptoregg" tag to each mudraptor egg
``` 
nameidentifier="mudraptoregg" Tags="smallitem,mudraptoregg,ignorebyai" 
```

Then in the petraptor recipe change it to
```
    <Fabricate suitablefabricators="medicalfabricator" requiredtime="10" requiresrecipe="true">
      <Item tag="mudraptoregg"/>
      <Item identifier="dementonite" />
    </Fabricate>
```

This is what it looks like
![image](https://user-images.githubusercontent.com/73229309/209069521-a637b78c-4f9a-4629-a468-64df04ba00c4.png)

### Version

0.20.16.1

**Comment by Tsunder** [comment]
aren't medium and large eggs usually un-pickupable? at least that's what happens when I feed them with saline? could this be a fault of the outpost eggs being pickable?

**Comment by ubertpendragon** [comment]
I remember not being able to pick up medium and large eggs, but now you can

**Comment by itchyOwl** [comment]
Fixed in Regalis11/Barotrauma-development@4767332. Initially it was intentional, because larger variants of mudraptor eggs were not pickable. When that changed, we didn't remember to update the petraptor egg.

**Comment by Rokvach** [comment]
Tested, working correctly. Closing.
